### ISSUE TITLE
Why does QGIS 3.10 tell me it is a development version (Windows 10)?

### ISSUE THREAD

**Issue by LeighB69** [qgis/QGIS#33494]
I installed QGIS 1.10 in October when it was released. I checked today to see if there was an update using "Check QGIS version" in the program and the response is that I'm "running a development version". 
Further: "We are working on 3.9-Master. It will go into feature freeze on 2019-08-30 12:00:00 UTC and be released as 3.10 on 2019-10-25 12:00:00 UTC. "
Is this a bug or have I somehow installed it incorrectly?

**Comment by roya0045** [comment]
Might just be a minor bug.

**Comment by LeighB69** [comment]
Thanks. I found there was an update 3.10.1 so will install this and check if it continues.

**Comment by LeighB69** [comment]
OK I installed 3.10.1, checked in the "about" that it over-wrote 3.10.0 but it still behaves the same. See attached screenshot.
![QGIS_Check](https://user-images.githubusercontent.com/54170169/71298971-a317d900-23c5-11ea-8b94-3c8df8866601.JPG)
![QGIS_About](https://user-images.githubusercontent.com/54170169/71298973-a57a3300-23c5-11ea-8842-eb9dd46e6259.JPG)

**Comment by agiudiceandrea** [comment]
It seems there is something not working properly in the new mechanism introduced with https://github.com/qgis/QGIS-Website/commit/e10533371f6f by @jef-n

While **[version.txt](https://version.qgis.org/version.txt)** is up to date:
```
#QGIS Version 31001|Visit https://download.qgis.org to get your copy of version 3.10.1|<html><body>
<ul>
<li>The current released version of QGIS is 3.10.1.
<li>We are working on 3.11-Master. It will go into feature freeze on 2020-01-17 12:00:00 UTC and be released as 3.12 on 2020-02-21 12:00:00 UTC.
<li>Details on changes can be found on github at <a href="https://github.com/qgis/QGIS/commits/master">https://github.com/qgis/QGIS/commits/master</a>.
</body>
</html>
```

the information in **[version-ltr.txt](https://version.qgis.org/version-ltr.txt)** are obsolete:
```
#QGIS Version 30411|Visit https://download.qgis.org to get your copy of version 3.4.11|<html><body>
<ul>
<li>The current available LTR version of QGIS is 3.4.11.
<li>The latest available version of QGIS is 3.8.2.
<li>We are working on 3.9-Master. It will go into feature freeze on 2019-08-30 12:00:00 UTC and be released as 3.10 on 2019-10-25 12:00:00 UTC.
<li>Details on changes can be found on github at <a href="https://github.com/qgis/QGIS/commits/master">https://github.com/qgis/QGIS/commits/master</a>.
</body>
</html>
```

Moreover, it seems the rewrite rule in the apache server configuration fails to properly identify LTR version by the QGIS User-Agent header.
The server sends version-ltr.txt at QGIS 3.10 requests, instead of sending version.txt.

**Comment by agiudiceandrea** [comment]
It seems there is something not working properly in the new mechanism introduced with https://github.com/qgis/QGIS-Website/commit/e10533371f6f (https://github.com/qgis/QGIS/pull/31298)

While **[version.txt](https://version.qgis.org/version.txt)** is up to date:
```
#QGIS Version 31001|Visit https://download.qgis.org to get your copy of version 3.10.1|<html><body>
<ul>
<li>The current released version of QGIS is 3.10.1.
<li>We are working on 3.11-Master. It will go into feature freeze on 2020-01-17 12:00:00 UTC and be released as 3.12 on 2020-02-21 12:00:00 UTC.
<li>Details on changes can be found on github at <a href="https://github.com/qgis/QGIS/commits/master">https://github.com/qgis/QGIS/commits/master</a>.
</body>
</html>
```

the information in **[version-ltr.txt](https://version.qgis.org/version-ltr.txt)** is obsolete:
```
#QGIS Version 30411|Visit https://download.qgis.org to get your copy of version 3.4.11|<html><body>
<ul>
<li>The current available LTR version of QGIS is 3.4.11.
<li>The latest available version of QGIS is 3.8.2.
<li>We are working on 3.9-Master. It will go into feature freeze on 2019-08-30 12:00:00 UTC and be released as 3.10 on 2019-10-25 12:00:00 UTC.
<li>Details on changes can be found on github at <a href="https://github.com/qgis/QGIS/commits/master">https://github.com/qgis/QGIS/commits/master</a>.
</body>
</html>
```

Moreover, it seems the rewrite rule in the apache server configuration fails to properly identify LTR version by the QGIS User-Agent header.
The server sends version-ltr.txt at QGIS 3.10 requests, instead of sending version.txt.
